src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import 'bulma/css/bulma.css';
 import './App.scss';
+import { useState } from 'react';
```

Comment:
Improve imports order 

Example:
```
import { useState } from 'react';
import 'bulma/css/bulma.css';

import './App.scss';
```

Useful link:
https://levelup.gitconnected.com/how-to-sort-imports-in-react-project-550f5ce70cbf